config.mak.uname

```diff
@@ -17,6 +17,13 @@ ifdef MSVC
 	uname_O := Windows
 endif
 
+
+ifdef MSVC
+GIT-MSVC-GEN: ./compat/vcbuild/find_vs_env.bat
+	@./compat/vcbuild/find_vs_env.bat | sed 's|\\|/|g' >GIT-MSVC-GEN
+-include GIT-MSVC-GEN
+endif
+
```

Comment:
Also: whose duty is it to generate `GIT-MSVC-GEN`? I *think* you want something like `Makefile` does for `GIT-VERSION` instead (`FORCE` as dependency, then test in the `.bat` whether it actually needs to run).